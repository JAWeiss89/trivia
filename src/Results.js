import React from 'react';
import './styles/Results.css'

const Results = ({score, name}) => {
    return (
        <div className="Results">
            <p className="Results-gameover">game over</p>
            <p className="Results-name">Great game, <span className="tandem">{name}</span>!</p>
            <p className="Results-score">Your score is {score}</p>

        </div>
    )
}

export default Results;