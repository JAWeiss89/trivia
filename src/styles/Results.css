.Results {
    width: 80%;
    height: 100%;
    position: relative;
    background-color: #ffffff;
    display: flex;
    flex-direction: column;
    justify-content: center;
    border-radius: 5px;
}

.Results-gameover {
    font-size: 5em;
    font-family: 'Roboto Mono', monospace;
}

.Results-name {
    font-size: 2.5em;
    
}

.Results-score {
    font-size: 1.7em;
}