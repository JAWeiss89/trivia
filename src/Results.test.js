import { render } from '@testing-library/react';
import Results from './Results';

// smoke test
test('renders learn react link', () => {
    render(<Results />);
});
  
// snapshot test
test("matches snapshot", () => {
const { asFragment } = render( <Results />);
expect(asFragment()).toMatchSnapshot();
});

test("screen shows with name", () => {
    const {queryByText} = render(<Results name="bob" score={12}/>);

    expect(queryByText("game over")).toBeInTheDocument();
    expect(queryByText("bob")).toBeInTheDocument();
})
  