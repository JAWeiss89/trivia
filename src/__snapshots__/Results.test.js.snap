// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Results"
  >
    <p
      class="Results-gameover"
    >
      game over
    </p>
    <p
      class="Results-name"
    >
      Great game, 
      <span
        class="tandem"
      />
      !
    </p>
    <p
      class="Results-score"
    >
      Your score is 
    </p>
  </div>
</DocumentFragment>
`;
